DependencyInjection/Configuration.php

```diff
@@ -59,12 +59,14 @@ public function getConfigTreeBuilder()
                 ->arrayNode('access_denied_listener')
                     ->canBeEnabled()
                     ->beforeNormalization()
-                        ->ifArray()->then(function ($v) { if (!empty($v) && empty($v['formats'])) {
-     unset($v['enabled']);
-     $v = ['enabled' => true, 'formats' => $v];
- }
+                        ->ifArray()->then(function ($v) {
+                            if (!empty($v) && empty($v['formats'])) {
+                                unset($v['enabled']);
```

Comment:
`$v['enabled']` is overridden at the next line, which is the reason for this `unset()` call?